deploy/kubernetes/webhook-example/patch-ca-bundle.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+# File originally from https://github.com/banzaicloud/admission-webhook-example/blob/blog/deployment/webhook-patch-ca-bundle.sh
```

Comment:
https://github.com/banzaicloud/admission-webhook-example does not exist and thus I can't check what license the script had :-(

Reply:
This comment is not addressed yet.

Reply:
I can see the comment is gone now.